hw/top_earlgrey/ip/pwrmgr/data/autogen/pwrmgr_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,39 @@
+// Copyright lowRISC contributors.
```

Comment:
quick question, i thought @matutem kept most of his test plans under `hw/ip/pwrmgr`, should those be moved here also?

Reply:
a lot of this is because the weird templating... but i should work on converting this to ip_autogen and all will be clearer.

Reply:
Yes. This is a duplicate of `hw/ip/<IP>/data/<IP>_sec_cm_testplan.hjson` which is also generated in this PR, where <IP>  is the list of IPs that are generated themselves. These toplevel specific ones are generated by `topgen` based on `<IP>.hjson.tpl` template, which I consider to be more accurate than the vanilla `<IP>.hjson`. See the vanilla `clkmgr.hjson` - it does not even list any countermeasures (but its `.tpl` counterpart does.

Reply:
Right, this whole autogen business needs reshuffling... later...

Reply:
sounds good. I think `pwr/rst/clk`mgrs really need to be shifted to `ip_autogen` so there is less ambiguity overall.
I will...do that....some day.